packages/node_modules/@webex/internal-plugin-ediscovery/test/unit/spec/content.js

```diff
@@ -195,6 +195,15 @@ describe('EDiscovery Content API Tests', () => {
 
       return result;
     });
+
+    it('GetContentSummary submits an audit log request', () => {
```

Comment:
Can you also add two more tests that demonstrate the error states?
* One to show you're catching a no report ID being given
* One to show what happens when the request fails (e.g. some HTTP Response other than 200)